### PR TITLE
add inspectable property for iOS/macOS

### PR DESCRIPTION
**Pull request by kevinvangelder**
As far as I can tell this *should* work, but does not.

https://developer.apple.com/documentation/webkit/wkwebview/4111163-inspectable/
https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/

### CODE REVIEW

**Comment by TheAlmightyBob** [comment]
What's the difference between this and [`webViewDebuggingEnabled`](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Reference.md#webviewDebuggingEnabled)?

File: example/ios/.xcode.env [hunk 1]
```diff
@@ -1 +1 @@
-export NODE_BINARY=/usr/local/bin/node
+export NODE_BINARY=/Users/kvg/.asdf/installs/nodejs/18.16.1/bin/node
```
Comment (o-alexandrov) [inline example/ios/.xcode.env:1]: You probably want to remove this leftover

**Review by o-alexandrov — CHANGES_REQUESTED** [review]
Example is broken

**Comment by kevinvangelder** [comment]
@TheAlmightyBob I looked for an existing implementation and could not find one, thanks for pointing it out.

**Pull request closed without merging**